bolt-sidecar/src/config.rs

```diff
@@ -27,6 +27,9 @@ pub struct Opts {
     /// Max commitments to accept per block
     #[clap(short = 'm', long)]
     pub(super) max_commitments: Option<usize>,
+    /// Validator indexes
+    #[clap(short = 'v', long)]
+    pub(super) validator_indexes: Option<String>,
```

Comment:
Clap has a feature called `value_parser` (docs [here](https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes) and [here](https://docs.rs/clap/latest/clap/struct.Arg.html#method.value_parser)) that's really cool.
I think that with this we could do without the `parse_validator_indexes` function

```suggestion
    /// Validator indexes
    #[clap(short = 'v', long, value_parser, num_args = 0.., value_delimiter = ',')]
    pub(super) validator_indexes: Vec<String>,
```

Reply:
ah good find again! I should have looked at the docs. Nit - I made the type as `Vec<u64>` (not string) as it can be parsed directly now

Reply:
Awesome!